docs/.vitepress/config/sidebar.ts

```diff
@@ -85,6 +85,7 @@ export const sidebar: DefaultTheme.Sidebar = {
       text: 'Tips and Tricks',
       collapsible: true,
       items:[
+        { text: 'Envd vs. other software', link:'/tips/comparison' },
```

Comment:
Could we place it in the last? After https://github.com/tensorchord/envd-docs/pull/90/files#diff-28b8d9c36ba8331d833f98071811f63570f78ce49c485f289c73aa5417aa931dL92